controlpanel/frontend/views/tool.py

```diff
@@ -51,21 +53,18 @@ class DeployTool(LoginRequiredMixin, RedirectView):
     def get_redirect_url(self, *args, **kwargs):
         name = self.kwargs["name"]
 
-        async_to_sync(channel_layer.send)(
-            'tools',
-            {
-                'type': 'deploytool',
-                'tool_name': name,
-                'user_id': self.request.user.id,
-            },
-        )
-        send_event(
-            'test', 'toolStatusChange', {'toolName': name, 'status': 'Deploying...'},
-        )
+        try:
+            Tool.create(name).deploy_for(self.request.user)
 
-        messages.success(
-            self.request, f"Deploying {name}... this may take up to 5 minutes",
-        )
+        except ToolDeploymentError as error:
+            messages.error(f"Failed deploying {name}")
```

Comment:
So this are flash messages, are they actually shown 😸? 🏆

Reply:
Yep
![Screenshot 2019-06-26 at 16 35 35](https://user-images.githubusercontent.com/35194/60193961-72008680-9830-11e9-8e05-01c08213d808.png)

Reply:
Wrong example, but you get the idea

Reply:
Awesome, I didn't remember we used this messages. Obviously there is something in the layout/template showing them 😆